releasenotes/notes/2.0/stabilizerstate-expval-sparsepauliop-3e32a871d8e908ce.yaml

```
@@ -1,5 +1,5 @@
 ---
 features_quantum_info:
   - |
-    The method :meth:`.StabilizerState.expectation_value` can now accept an operator of type
+    The method :meth:`.StabilizerState.expectation_value` method can now accept an operator of type
```

"method" is mentioned twice
```suggestion
    The :meth:`.StabilizerState.expectation_value` method can now accept an operator of type
```